io-engine/src/grpc/v1/replica.rs

```diff
@@ -253,6 +254,9 @@ impl ReplicaRpc for ReplicaService {
                     }
                     Ok(lvol) => {
                         debug!("created lvol {:?}", lvol);
+                        if let Some(v) = &args.entity_id {
+                            let _ = lvol.set_blob_attr("entity_id", v.to_string(), true).await;
```

Comment:
If set_blob_attr is failed, how control-plane will know about this error. I think error should be return back from dataplane which can be used in control-plane to mark the replica creation fail.

Reply:
As We are sending back Option<entity_id> in the response, I think it is okay to mark the operation success but are We planning to do any action if entity_id is set to None in the create replica request or It will use the `set_replica_entity_id` grpc only during share replica flow ?

Reply:
I think we should fail the operation, because clearly the expectation that the replica has entity id is not met. What do you expect the control-plane to do?